message.go

```diff
@@ -1130,6 +1130,28 @@ func (m *RedisMessage) AsGeosearch() ([]GeoLocation, error) {
 	return geoLocations, nil
 }
 
+// AsAnyAnyMap converts RedisMessage to map[any]any.
+// we don't use (*RedisMessage).ToAny() on map type because it will convert ele
+// to map[string]any
+func (m *RedisMessage) AsAnyAnyMap() (map[any]any, error) {
```

Comment:
`map[any]any` is too hard to use. I don't think we should have this function.

Reply:
Okay, I'll remove it.